src/engine/client/backend_sdl.cpp

```diff
@@ -269,7 +269,7 @@ bool CCommandProcessorFragment_SDL::RunCommand(const CCommandBuffer::SCommand *p
 
 // ------------ CCommandProcessor_SDL_GL
 
-void CCommandProcessor_SDL_GL::HandleError()
+void CCommandProcessorSDLGL::HandleError()
```

Comment:
```suggestion
void CCommandProcessorSdlGl::HandleError()
```
Also the comment above.